manifests/config.pp

```diff
@@ -70,5 +97,25 @@
       notify  => Class['confluence::service'],
     }
 
+    file { "${confluence::webappdir}/confluence/WEB-INF/web.xml":
+      source  => 'puppet:///modules/confluence/web.xml',
+      mode    => '0600',
+      owner   => $::confluence::user,
+      group   => $::confluence::group,
+      require => Class['confluence::install'],
+      notify  => Class['confluence::service'],
+    }
+
+  }
+  # if JDBC was configured along with the license key and server_id, skip some server setup steps.
+  if $confluence::tomcat_jdbc_settings and $confluence::license and $confluence::server_id {
+    file { "${confluence::homedir}/confluence.cfg.xml":
```

Comment:
Right so the users configure the main class either via direct parameters or Hiera data, but, for whatever reason, this config class already had params rather than just using them from the main class, so I followed the pattern when adding my changes.